mongo/integration/initial_dns_seedlist_discovery_test.go

```diff
@@ -63,6 +73,24 @@ func runSeedlistDiscoveryDirectory(mt *mtest.T, subdirectory string) {
 	}
 }
 
+// runSeedListDiscoverPingTest will create a new connection using the test URI and attempt to "ping" the server.
+func runSeedListDiscoveryPingTest(mt *mtest.T, clientOpts *options.ClientOptions) {
```

Comment:
```suggestion
func runSeedlistDiscoveryPingTest(mt *mtest.T, clientOpts *options.ClientOptions) {
```
[nit] we should use the same case as `runSeedlistDiscoveryTest`.